### DISCUSSION TITLE
Components Roadmap for Primitives?

### DISCUSSION THREAD

**Discussion by XMethues** [radix-ui/primitives#929] [Ideas] [answered]
Is there a components roadmap for primitives? I mean more components and useful hooks. Like Adobe's react-spectrum does.

**Comment by colmtuite** [comment, marked as answer]
Hey, we don't have one currently. We're releasing a new website very soon, and 3 new primitives shortly after, so we'll take a look at publishing a roadmap after that. Great idea, I'd love a public roadmap.
